apps/crn-server/test/integration/common/reminders.integration-test.ts

```diff
@@ -600,7 +604,8 @@ describe('Reminders', () => {
           .put(`/research-outputs/${draftOutputId}`)
           .send({
             ...draftOutput,
-            reviewRequestedById: loggedInUser.id,
+            statusChangedById: loggedInUser.id,
+            isInReview: true,
```

Comment:
Rather than adding this everywhere - if it isn't pertinent to the test - then I'd add it to the default data [here](https://github.com/yldio/asap-hub/blob/master/apps/crn-server/test/integration/fixtures/research-output.ts#L23-L41).

Reply:
the default value for this field is false, so needs to be overwritten for these tests